content/en/docs/faq/_index.md

```diff
@@ -424,9 +424,11 @@ you explicitly specify `--create-namespace` flag.
 
 ### Why aren't there native packages of Helm for Fedora and other Linux distros?
 
-We'd love to provide these or point you toward a trusted provider. If you're
-interested in helping, we'd love it. This is how the Homebrew formula was
-started.
+The Helm project does not maintain packages for operating systems and
+environments. The Helm community many provide native packages and if the Helm
```

Comment:
may provide? provides many?

Reply:
oops. Thanks for catching that. Fixed that small slip.